.github/workflows/build_and_release.yml

```
@@ -0,0 +1,28 @@
+name: Build And Deploy App to Firebase Distribution
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build_and_release_job:
+    name: Build and release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Make gradlew executable
+        run: chmod +x ./gradlew
+      - name: build debug
+        run: ./gradlew assembleDevDebug
```

The release build should be the `release` build type. 😄 
```suggestion
        run: ./gradlew assembleDevRelease
```